manoseimas/mps_v2/migrations/0001_initial.py

```diff
@@ -2,6 +2,8 @@
 from __future__ import unicode_literals
 
 from django.db import models, migrations
+import autoslug.fields
+import manoseimas.mps_v2.models
```

Comment:
Yes, migrations make it more difficult to rename apps, that's where having `core` app helps a lot.